doc/topics/development/deprecations.rst

```diff
@@ -17,17 +17,17 @@ new behavior, the deprecation time frame should be carefully determined based
 on the significance and complexity of the changes required by the user.
 
 Salt feature releases are based on the Periodic Table. Any new features going
-into the develop branch will be named after the next element in the Periodic
-Table. For example, Beryllium was the feature release name of the develop
-branch before the 2015.8 branch was tagged. At that point in time, any new
-features going into the develop branch after 2015.8 was branched were part of
-the Boron feature release.
+into the ``master`` branch will be named after the next element in the Periodic
+Table. For example, Magnesium was the feature release name of the develop
+branch before the ``3002`` branch was tagged. At that point in time, any new
+features going into the ``master`` branch after ``3002`` was branched were part of
+the Aluminium feature release.
```

Comment:
We no longer branch for a release, but release based on a tag. We likely can take out all of this wording. For the purpose of deprecations this was significant detail when we did maintain release branches.

Reply:
Made some modifications to the wording. How's it look?